src/app/components/core/control-dispatcher/api-flow/api-flow.component.html

```diff
@@ -12,10 +12,12 @@
                 <p *ngIf="viewPort !== 'mobile'" class="add-btn">Add &nbsp;</p>
                 <i class="fa fa-xs fa-plus" aria-hidden="true"></i>
             </a>
-            <!-- <span class="icon is-small tooltip has-tooltip-bottom has-tooltip-arrow is-pulled-right is-hovered btn-margin"
-                data-tooltip="Help" (click)="goToLink('control-dispatcher-service')">
-                <i class="far fa-question-circle"></i>
-            </span> -->
+            <a *ngIf="isServiceAvailable && rolesService.hasControlAccess()" class="button cog-icon"
+                routerLink="/settings/manage-services" id="config-icon">
+                <span class="icon">
+                <i class="fa fa-sm fa-cog" aria-hidden="true" title="Dispatcher Service Configuration"></i>
+                </span>
+            </a>
```

Comment:
This is a duplicate code block used in many files, better to move it in separate child component.